geps/gep-1709.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1709: Conformance Profiles
 
 * Issue: [#1709](https://github.com/kubernetes-sigs/gateway-api/issues/1709)
-* Status: Provisional
+* Status: Prototyping
```

Comment:
This needs an update to `mkdocs.yml`.

Reply:
Sounds good! 249f38800c0624e80575dc450a91fff3bf259ff5 resolves